.github/workflows/add-fleet-issues-to-ingest-project.yml

```diff
@@ -1,14 +1,14 @@
-name: Add Fleet issue to Platform Ingest project
+name: Add Fleet issue to Platform Ingest - UI & Integrations project
 
 on:
   issues:
     types:
       - labeled
       
 env:
-  INGEST_PROJECT_ID: 'PVT_kwDOAGc3Zs4AEzn4'
+  INGEST_PROJECT_ID: 'PVT_kwDOAGc3Zs4AogrP'
   FLEET_LABEL: 'Team:Fleet'
-  AREA_FIELD_ID: 'PVTSSF_lADOAGc3Zs4AEzn4zgEgZSo'
+  AREA_FIELD_ID: 'PVTSSF_lADOAGc3Zs4AogrPzggEiBs'
   FLEET_UI_OPTION_ID: '411a7b86'
```

Comment:
The option ID's actually remain consistent across cloned projects, so this didn't need to be updated, e.g. 

```
{
  "id": "PVTSSF_lADOAGc3Zs4AogrPzggEiBs",
  "name": "Area",
  "options": [
    {
      "id": "b85d422b",
      "name": "Ecosystem"
    },
    {
      "id": "411a7b86",
      "name": "Fleet UI"
    },
...
```